docs/docusaurus/.env.example

```diff
@@ -0,0 +1 @@
+POSTHOG_API_KEY=
```

Comment:
I don't think we need this anymore. can you instead add the instructions in the README that we should include the POSTHOG_API_KEY from our PostHog dev env in the run script? 

e.g. To test the `PostHog-Docusaurus` plugin, simply include the env var when running docusaurus locally: `start": "POSTHOG_API_KEY=BLABLABLA docusaurus start"`